src/java/htsjdk/tribble/dbsnp/OldDbSNPCodec.java

```diff
@@ -89,6 +89,11 @@ public OldDbSNPFeature decode(String line) {
     }
 
     @Override
+    public boolean canDecode(String path){
+	return path.toLowerCase().endsWith(".dbsnp") || path.toLowerCase().endsWith(".rod") || path.toLowerCase().endsWith(".snp");
```

Comment:
This codec should either be deleted (based on @vdauwera 's comment above), or not claim generic extensions such as `.rod` and `.snp` in its `canDecode()` method.